src/main/java/org/broadinstitute/hellbender/tools/walkers/mutect/M2FiltersArgumentCollection.java

```diff
@@ -30,25 +30,25 @@
     @Argument(fullName = "max_germline_posterior", optional = true, doc = "Maximum posterior probability that an allele is a germline variant")
     public double maxGermlinePosterior = 0.025;
 
-    @Argument(fullName = "maxAltAllelesThreshold", optional = true, doc="filter variants with too many alt alleles")
+    @Argument(fullName = "maxAltAllelesThreshold", optional = true, doc = "filter variants with too many alt alleles")
```

Comment:
Hate to do this, can you make these part of a separate commit.  It's best not to modify lines that aren't part of creating this filter.